src/ostree/ot-admin-builtin-status.c

```diff
@@ -30,13 +30,21 @@
 
 static gboolean opt_verify;
 static gboolean opt_skip_signatures;
-
-static GOptionEntry options[]
-    = { { "verify", 'V', 0, G_OPTION_ARG_NONE, &opt_verify, "Print the commit verification status",
-          NULL },
-        { "skip-signatures", 'S', 0, G_OPTION_ARG_NONE, &opt_skip_signatures,
-          "Print the commit verification status", NULL },
-        { NULL } };
+static gboolean opt_query_booted;
+
+static GOptionEntry options[] = {
+  { "verify", 'V', 0, G_OPTION_ARG_NONE, &opt_verify, "Print the commit verification status",
+    NULL },
+  { "skip-signatures", 'S', 0, G_OPTION_ARG_NONE, &opt_skip_signatures, "Skip signatures in output",
+    NULL },
+  { "query-booted", 'Q', 0, G_OPTION_ARG_NONE, &opt_query_booted,
```

Comment:
This is pure bikeshed, but I think this would be better named e.g. `--is-default`. Somewhat precedence are the `systemctl is-*` commands that also print strings. `--query-booted` is a bit ambiguous.